### ISSUE TITLE
[Crash] World load causes crash due to Earth2Java's Pinto Cow, involving Elytra Slot?

### ISSUE THREAD

**Issue by purejosh** [TheIllusiveC4/CuriousElytra#75]
### Minecraft Version

1.19.2

### What happened?

Hey, got a weird one. I'm seeing in the stacktrace that it's referencing Elytra Slot for the crash, but the crash itself is from rendering an entity, and that entity is the earth2java's pinto_cow... It's weird. Can you take a look at this, please? I can't tell who to ask here, I don't usually see two mods referenced like this. 

### How do you trigger this bug?

Using the modpack I'm using, it happens on world load. 

### Loader

Fabric

### Loader Version

Fabric 0.14.19

### API Version

0.76.0

### Mod Version

6.1.1+1.19.2

### Relevant Log Outputs

https://gist.github.com/purejosh/07027beda74a07a659f93567c1204a4e

**Comment by TheIllusiveC4** [comment]
That's a strange bug. Does it happen on every world load? Does restarting help? What about redownloading the Elytra Slot mod?

**Comment by purejosh** [comment]
Just closed my client, reinstalled Elytra Slot using Curseforge, and created a new world, same crash but different entity?? lol this makes no sense... 
https://gist.github.com/purejosh/f8d2cd127e0756b2a4161bbd844f1152

**Comment by purejosh** [comment]
A minecraft entity, no less.

**Comment by purejosh** [comment]
Just tried 6.1.0, no luck. This might be an incompat with a new mod I've added recently. 
![image](https://github.com/illusivesoulworks/elytraslot/assets/70633949/48ef13ee-d89a-4414-b423-21b90b096918)

**Comment by TheIllusiveC4** [comment]
Do you have a list of the new mods you added that may have caused this to begin occurring?

**Comment by purejosh** [comment]
Yeah, the screenshot I posted above are all the new mods I've either added or updated today.

**Comment by purejosh** [comment]
Selectively removing mods and testing now.

**Comment by purejosh** [comment]
I've removed all of the mods I've added today and it's still crashing. This is very weird.

**Comment by purejosh** [comment]
Alright, I removed https://www.curseforge.com/minecraft/mc-mods/connectivity, and somehow, now it's working. 

I went down the rabbit hole, and as soon as I removed the mod that the crash said was responsible, it would just change to another mod, including my own mods... as soon as I saw that behavior, I just started removing a handful of mods at a time, and finally found the culprit. 

Thanks for your time.
